code/modules/jobs/job_types/chaplain.dm

```diff
@@ -1,96 +1,96 @@
+	minimal_access = list(ACCESS_MORGUE, ACCESS_CHAPEL_OFFICE, ACCESS_CREMATORIUM, ACCESS_THEATRE)
+	paycheck = PAYCHECK_EASY
+	paycheck_department = ACCOUNT_CIV
+
+	display_order = JOB_DISPLAY_ORDER_CHAPLAIN
+
+
+/datum/job/chaplain/after_spawn(mob/living/H, mob/M)
+	. = ..()
+	if(H.mind)
+		H.mind.isholy = TRUE
+
+	var/obj/item/storage/book/bible/booze/B = new
+
+	if(SSreligion.religion)
+		B.deity_name = SSreligion.deity
+		B.name = SSreligion.bible_name
+		B.icon_state = SSreligion.bible_icon_state
+		B.item_state = SSreligion.bible_item_state
+		to_chat(H, "There is already an established religion onboard the station. You are an acolyte of [SSreligion.deity]. Defer to the Chaplain.")
+		H.equip_to_slot_or_del(B, SLOT_IN_BACKPACK)
+		var/nrt = SSreligion.holy_weapon_type || /obj/item/nullrod
+		var/obj/item/nullrod/N = new nrt(H)
+		H.put_in_hands(N)
+		return
+
+	var/new_religion = DEFAULT_RELIGION
+	if(M.client && M.client.prefs.custom_names["religion"])
+		new_religion = M.client.prefs.custom_names["religion"]
+
+	var/new_deity = DEFAULT_DEITY
+	if(M.client && M.client.prefs.custom_names["deity"])
+		new_deity = M.client.prefs.custom_names["deity"]
+
+	B.deity_name = new_deity
+
+
+	switch(lowertext(new_religion))
+		if("christianity") // DEFAULT_RELIGION
+			B.name = pick("The Holy Bible","The Dead Sea Scrolls")
+		if("satanism")
+			B.name = "The Unholy Bible"
+		if("cthulhu")
+			B.name = "The Necronomicon"
+		if("islam")
+			B.name = "Quran"
+		if("scientology")
+			B.name = pick("The Biography of L. Ron Hubbard","Dianetics")
+		if("chaos")
+			B.name = "The Book of Lorgar"
+		if("imperium")
+			B.name = "Uplifting Primer"
+		if("toolboxia")
+			B.name = "Toolbox Manifesto"
+		if("homosexuality")
+			B.name = "Guys Gone Wild"
+		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "meme", "memes")
+			B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition")
+			H.adjustBrainLoss(100) // starts off retarded as fuck
+		if("science")
+			B.name = pick("Principle of Relativity", "Quantum Enigma: Physics Encounters Consciousness", "Programming the Universe", "Quantum Physics and Theology", "String Theory for Dummies", "How To: Build Your Own Warp Drive", "The Mysteries of Bluespace", "Playing God: Collector's Edition")
+		else
+			B.name = "The Holy Book of [new_religion]"
+
+	SSreligion.religion = new_religion
+	SSreligion.bible_name = B.name
+	SSreligion.deity = B.deity_name
+
+	H.equip_to_slot_or_del(B, SLOT_IN_BACKPACK)
+
+	SSblackbox.record_feedback("text", "religion_name", 1, "[new_religion]", 1)
+	SSblackbox.record_feedback("text", "religion_deity", 1, "[new_deity]", 1)
+
+/datum/outfit/job/chaplain
+	name = "Chaplain"
+	jobtype = /datum/job/chaplain
+
+	belt = /obj/item/pda/chaplain
+	uniform = /obj/item/clothing/under/rank/chaplain
+	backpack_contents = list(/obj/item/camera/spooky = 1)
+	backpack = /obj/item/storage/backpack/cultpack
```

Comment:
What exactly did you change here?

Reply:
may have accidently ruined the changes with carriage return line feed memes, nothing is different that i can see